publish-extension.js

```diff
@@ -103,7 +103,7 @@ const { createVSIX } = require('vsce');
             }
         }
         // TODO(ak) check license is open-source
-        if (!manifest?.license && !(packagePath && await ovsx.isLicenseOk(packagePath, manifest))) {
+        if (!xmlManifest?.PackageManifest?.Metadata[0]?.License && !(packagePath && await ovsx.isLicenseOk(packagePath, manifest))) {
```

Comment:
License [is an array ](https://github.com/microsoft/vscode-vsce/blob/228d0e1fc35031786d1fa436e05cb091ffa8a430/src/xml.ts#L17)so better check if it has at least one element 
```suggestion
        if (!xmlManifest?.PackageManifest?.Metadata[0]?.License?.[0] && !(packagePath && await ovsx.isLicenseOk(packagePath, manifest))) {
```

Reply:
Thanks!